src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/WasmTestCommand.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.DotNet.XHarness.CLI.Commands.Wasm
 {
     internal class WasmTestCommand : TestCommand
     {
-        private const string CommandHelp = "Executes tests on WASM using a selected JavaScript engine";
+        private const string CommandHelp = "Executes tests on WASM using a selected JavaScript engine or a browser";
```

Comment:
I wonder whether it would make sense to split this off from the `wasm test` command into e.g. `wasm test-browser` since the features are pretty different.

@premun thoughts?

Reply:
Yeah, if we have a completely different code handling this then it shouldn't be difficult and I would be up for it.
We then wouldn't need the Chrome engine type which makes sense because it's not an engine.